### ISSUE TITLE
cat ../k8s_deployment.tf | cdktf convert  --provider=kubernetes

### ISSUE THREAD

**Issue by arikamir** [hashicorp/learn-terraform-cdktf-applications#5]
Hi
I am following the tutorial, and `cdktf convert` fails with:
```
TypeError: Cannot read properties of undefined (reading 'push')
    at Object.VWr (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:662:2616)
```

**Comment by DanielMSchmidt** [comment]
Hey @arikamir thanks for reporting this bug 👍  It's fixed now and released in cdktf 0.9.3 👍 a quick `npm install -g cdktf-cli@latest` should fix it

**Comment by im2nguyen** [comment]
Thanks @DanielMSchmidt! Going to close this issue for now 😄
